documents/Code of Ethics.md

```diff
@@ -1,87 +1,97 @@
          1. Example: Contacting sponsors for a competition should use personal email addresses.
-   3. WCA Delegates must communicate with other Delegates in their region and/or in any region they are planning to Delegate a competition.
-   4. All work done within the WCA is confidential and must not be disclosed to non-WCA Staff members without explicit permission from relevant members.
+   2. WCA Delegates must communicate with other Delegates in their region and/or in any region they are planning to delegate a competition.
+   3. All work done within the WCA is confidential and must not be disclosed to non-WCA Staff members without explicit permission from relevant members.
+      1. Delegates may provide relevant feedback to the Organizers but must not give visibility of incidents, or sections written by other Delegates.
 3. **Conduct of Official WCA Competitions**
-   1. Decisions on incidents required to be made by a WCA Delegate must only be made by those officially listed on the competition page.
+   1. Decisions on incidents required to be made by a WCA Delegate must only be made by those listed on the WCA competition page.
       1. Non-listed Delegates can make decisions on incidents only with the approval of the listed Delegate(s).
-   2. Decisions on competition changes should be approved by both organizer(s) and listed Delegate(s) through open, joint discussions. This includes decisions before, during, and after the competition.
-   3. WCA Delegates should work in conjunction with the organizer(s) to hold the events requested by the organizer(s).
-      1. Possible reasons for declining organizers’ requests include:
+   2. Decisions on competition changes should be approved by both Organizer(s) and listed Delegate(s) after discussion. This includes decisions before, during, and after the competition.
+   3. WCA Delegates should work in conjunction with the Organizer(s) to hold the events requested by the Organizer(s).
+      1. Possible reasons for declining Organizers’ requests include:
          1. Scheduling issues
          2. Lack of equipment to hold an event
          3. Event frequency in the area
          4. Inexperience of the community
-         5. Inexperience of the organizer
-         6. Past problems with an event held by the organizer/community
-      2. Delegates should work alongside the organizer(s) to plan and resolve any of these reasons listed under 3.3.1.
-   4. Delegates and Organizers should only be officially listed for a competition if they perform a necessary role in the running of the event.
+         5. Inexperience of the Organizer
+         6. Past problems with an event held by the Organizer/community
+      2. Delegates should work alongside the Organizer(s) to plan and resolve any of these reasons listed under 3.3.1.
+   4. Delegates and Organizers should only be officially listed for a competition if they perform a significant role.

```

Comment:
What is a "significant role" - I feel it should be defined? I would list a Delegate if they are at the competition and handle a minor incident even to the trivial nature of "yes, that is a DNF". Obviously per the regulations even the granting of a basic extra attempt must be done by a delegate, as with handling of scrambles, which would necessitate listing the delegate, even if they did not meet the "significant role" barrier.

Reply:
Some people had issues with the word "necessary" in the previous version. Granting an extra seems significant to me, given that you have to be a WCA Delegate in order to grant one. (I.e. if they require listing because of an action they took -> that action was significant)

There are a million and one things that can happen at competitions to Delegates, I don't think that turning the Code of Ethics into the Regulations where we try and cover every possible scenario is feasible.

Reply:
If we are not being prescriptive about what "significant" is then I don't see the point having it there. I actually thing a broader problem is Delegates _not_ being listed even when they were at the competition and staffing/helping out such they were perceived as listed delegates (even if they never granted an extra).

Reply:
I can't imagine how we can cover all cases Delegates can encounter and deal with, therefore wording such as "significant" (previously "necessary") is required to avoid attempting to write an endless list which would still leave holes. Granted, some perceptions of "significant" may be different than others, but we already see this in the regulations where being prescriptive is the name of the game.

Reply:
I see your point, and I think "significant" is better wording than "necessary". I think we should make it clear when messaging these changes to Delegates that any task defined in the regulations as a task for the Delegate meets the criteria of significant.

Reply:
That's fair enough, I will outline that in the announcement to Staff when it's sent! :-)